app/models/user.rb

```diff
@@ -20,10 +22,10 @@ class User < ActiveRecord::Base
 
   has_attached_file :uploaded_avatar,
     styles: {
-              large: "#{User::LARGE_IMAGE}x#{User::LARGE_IMAGE}#",
-              medlarge: "#{User::MED_LARGE_IMAGE}x#{User::MED_LARGE_IMAGE}#",
-              medium: "#{User::MEDIUM_IMAGE}x#{User::MEDIUM_IMAGE}#",
-              small: "#{User::SMALL_IMAGE}x#{User::SMALL_IMAGE}#",
+              large:    "#{IMAGE_SIZES.fetch(:large, :medium)}x#{IMAGE_SIZES.fetch(:large, :medium)}",
+              medlarge: "#{IMAGE_SIZES.fetch(:medlarge, :medium)}x#{IMAGE_SIZES.fetch(:medlarge, :medium)}",
+              medium:   "#{IMAGE_SIZES.fetch(:medium)}x#{IMAGE_SIZES.fetch(:medium)}",
+              small:    "#{IMAGE_SIZES.fetch(:small, :medium)}x#{IMAGE_SIZES.fetch(:small, :medium)}"
```

Comment:
this is not pretty 
I tried making a class method, but that was also ugly :fish_cake: